pkg/streamingpromql/functions/common_test.go

```diff
@@ -81,7 +88,8 @@ func TestFloatTransformationDropHistogramsFunc(t *testing.T) {
 		Histograms: nil, // Histograms should be dropped
 	}
 
-	modifiedSeriesData, err := transformFunc(seriesData, limiting.NewMemoryConsumptionTracker(0, nil))
+	modifiedSeriesData, err := transformFunc(seriesData, memoryConsumptionTracker)
 	require.NoError(t, err)
 	require.Equal(t, expected, modifiedSeriesData)
+	require.Equal(t, types.FPointSize*2, memoryConsumptionTracker.CurrentEstimatedMemoryConsumptionBytes)
```

Comment:
Would be good to add a comment here explaining that we expect the histogram slice to have been returned to the pool.